client/src/components/Sub_ProductDetail/AddToCart.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import SizeSelector from './Sub_AddToCart/SizeSelector.jsx';
+import QuantitySelector from './Sub_AddToCart/QuantitySelector.jsx';
+
+function AddToCart({ skus, startSelect, currentStyle }) {
+  const [skuID, setSkuID] = useState();
+  const [quantity, setQuantity] = useState(1);
+  const [showAlert, setShowAlert] = useState(false);
+  const [validQuantities, setValidQuantities] = useState([1]);
+  const validSkus = Object.entries(skus).filter((entry) => entry[1].quantity > 0);
+
+  const handleSize = (e) => { setSkuID(e.target.value); setQuantity(1); };
+
+  const handleQuantity = (e) => setQuantity(e.target.value);
+
+  const submitCart = () => {
+    const config = {
+      sku_id: skuID,
+      count: quantity,
+    };
+    axios.post('/api/cart', config)
+      .then()
```

Comment:
I didn't realize you could just have an empty .then block to resolve the promise. That's really cool.

Reply:
oh i simply forgot to fill in the parenthesis... somehow it works so i'll leave it as is...